WORKSPACE

```diff
@@ -421,7 +421,7 @@ go_repository(
 ##
 
 # Change this and the docker/Dockerfile.proxy* files together
```

Comment:
maybe also add here that we currently get the sha from the jenkins post submit ?